scripts/deploy_on_kind.sh

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env bash
+
+set -e
+
+MR_NAMESPACE="${MR_NAMESPACE:-kubeflow}"
+
+# modularity to allow re-use this script against a remote k8s cluster
+if [[ -n "$LOCAL" ]]; then
+    CLUSTER_NAME="${CLUSTER_NAME:-kind}"
+    IMG="${IMG:-docker.io/kubeflow/model-registry:main}"
```

Comment:
```suggestion
    IMG="${IMG:-localhost/kubeflow/model-registry:latest}"
```

I had to make this change locally to make e2e tests work, screenshot of the error:

![image](https://github.com/user-attachments/assets/6c14cd5c-1d92-4430-92b2-d177393aacef)


after the change:

![image](https://github.com/user-attachments/assets/99a5c37e-89e0-4e52-a2fe-315f477ce43f)